### DISCUSSION TITLE
List of Permission types

### DISCUSSION THREAD

**Discussion by satoshinishimura-fc** [iTwin/itwinjs-core#3049] [Q&A] [answered]
Anyone knows whete I can access to a list of permission types and their definitions. Thanks.

I took a look on [this page](https://developer.bentley.com/apis/projects/operations/get-user-permissions/) but I could not find a list of permission types.
If such list with the definition of the permission exists, it is helpful for me to manage user roles.

**Comment by davidhjones** [comment, marked as answer]
@satoshinishimura-fc each API endpoint documented on developer.bentley.com should have an Authorization section, describing the necessary permissions for that endpoint. You can see an example documented here: https://developer.bentley.com/apis/imodels/operations/create-imodel/#authorization. Unfortunately, we do not have a single page which describes the exhaustive list of available permissions.

Can you provide more details on what workflow you're trying to support? How are you utilizing user-roles in your product and what process if there to manage these roles?

**Reply by satoshinishimura-fc** [reply]
Thank you for your prompt reply.
> Unfortunately, we do not have a single page which describes the exhaustive list of available permissions.

I understood.

> Can you provide more details on what workflow you're trying to support? How are you utilizing user-roles in your product and what process if there to manage these roles?

Sorry, I have no concrete idea but I was just curious. When I get how to utilize user-roles more concrete then I will consult you here again.
Thank you for your support.
